runtime/interpreter/encoding_test.go

```diff
@@ -3097,7 +3097,7 @@ func TestEncodeDecodeCapabilityValue(t *testing.T) {
 			// positive integer to follow
 			0x18,
 			// public account (tag)
-			0x5b,
+			0x59,
```

Comment:
This change seems suspicious, I'll have a look

Reply:
We have a test at the end of `statictype_test.go` asserting that there are exactly 98 static types (why?), so in order to add the two new inbox types at the end, I had to move the other account static types two earlier in the list, hence this number decreasing by two. If you'd rather change that test to account for 100 types instead, then that is also an option.